ccm_web/server/src/api/api.interfaces.ts

```diff
@@ -8,12 +8,19 @@ export interface Globals {
     roles: string[]
   }
 }
-
-export interface APIErrorData {
+export interface APIErrorHandler {
```

Comment:
```suggestion

export interface APIErrorHandler {
```

Reply:
I suggested combining this type with m`APIErrorPayload` elsewhere.

`Handler` sounds odd to me as a data type, I would expect it to be class.